### PR TITLE
Move `external-services` to apps

### PR DESCRIPTION
**Pull request by kachkaev**
## 🌟 What is the purpose of this PR?

This PR moves `packages/hash/external-services` to `apps/hash-external-services` as part of repo restructuring.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203543021352032/1203324923411371/f) _(internal)_

## ⚠️ Known issues

- The repo still contains `packages/hash/README.md` with references to external services. I have updated the path, but we might need to move this README elsewhere and review its contents.

## 🐾 Next steps

- Move `packages/hash/*` apps to `apps`
- Move `packages/hash/graph` and `packages/hash/libs/error-stack` to libs
- Move `packages/hash/docker` to `infra/docker`, then spread `Dockerfile`s across workspaces
- [discuss] Split `external-services` into individual modular apps - [see this Slack thread](https://hashintel.slack.com/archives/C022217GAHF/p1674056552073489?thread_ts=1674037284.832079&cid=C022217GAHF) (_internal link_)

## 🛡 What tests cover this?

- CI

### CODE REVIEW

**Review by thehabbos007 — APPROVED** [review]
Thank you, Alex! I tested this locally with normal & prod images, and it seems to work fine. I also went through some of the graph-specific makefile rules we have to make sure we aren't incorrectly referring to an old docker compose, which we don't seem to.

I just have a single nit, other than that we need to update some internal docs referencing the production docker compose location (I'll make sure to do so.)

File: apps/hash-external-services/package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@apps/hash-external-services",
+  "version": "0.0.0-private",
+  "private": true,
+  "description": "External services needed to run HASH (database, graph, etc.)"
```
[the remark this answers is not in the document]
Comment (kachkaev) [reply apps/hash-external-services/package.json:5]: Thanks @thehabbos007! I’ll update the description as part of #1879 when this PR is merged!

**Review by nonparibus — APPROVED** [review]
Legal ✅

File: apps/hash-external-services/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (kachkaev) [reply apps/hash-external-services/package.json:5]: https://github.com/hashintel/hash/pull/1879/commits/e00cb1b8d7ed61f7a3391452d351cb5fcd0e099a

**Pull request merged**